src/arch_mips.py

```diff
@@ -0,0 +1,480 @@
+from .types import Type
+from .parse_instruction import Register
+from .translate import (
+    Arch,
+    BinaryOp,
+    CmpInstrMap,
+    CommentStmt,
+    ErrorExpr,
+    Expression,
+    InstrMap,
+    InstrSet,
+    Literal,
+    PairInstrMap,
+    StmtInstrMap,
+    StoreInstrMap,
+    UnaryOp,
+    as_f32,
+    as_f64,
+    as_int64,
+    as_intish,
+    as_intptr,
+    as_ptr,
+    as_s32,
+    as_s64,
+    as_type,
+    as_u32,
+    as_u64,
```

Comment:
I think it might be a good follow-up to reduce imports by grouping these together in classes or megafunctions like `as()` which takes in an enum. 

Same for the `handle_` family.

Reply:
Yeah I'd be more inclined to use a class as a namespacing technique -- the `as_` fns are already mostly very thin wrappers around `as_type()`.